cloud/gce_util.go

```diff
@@ -98,22 +97,6 @@ func makeImage(name string) string {
 	return fmt.Sprintf("global/images/%s", name)
 }
 
-// Generates a unique instance name for an instance based on the distro ID.
-// Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)'
-func generateName(d distro.Distro) string {
```

Comment:
I kind of worry that we should try and move this into the distro.GenerateName() method and that this will break things if we try to actually use GCP